src/account-info/ProfileScreen.js

```diff
@@ -29,6 +30,17 @@ const styles = createStyleSheet({
     flex: 1,
     margin: 8,
   },
+  deactivateText: {
+    color: '#e5898e',
```

Comment:
We write colors as HSL, not RGB. It looks like this isn't in the style guide (I guess we should add it there!); but you can read about it at #3516 which was the issue where we converted to HSL from a mixture of HSL and RGB.